src/artm/messages.proto

```diff
@@ -846,3 +846,8 @@ message TransformMasterModelArgs {
   optional ThetaMatrixType theta_matrix_type = 3 [default = Dense];
   optional string predict_class_id = 4;
 }
+
+message ConfigureLoggingArgs {
```

Comment:
But this enum can be used, if levels will be smth like 'Low', 'Medium', 'High' and 'All'

14.02.2016, 01:30, "sashafrey" <notifications@github.com>:
> In src/artm/messages.proto:
>
>> @@ -846,3 +846,8 @@ message TransformMasterModelArgs { > optional ThetaMatrixType theta_matrix_type = 3 [default = Dense]; > optional string predict_class_id = 4; > } > + > +message ConfigureLoggingArgs {
>
> message ConfigureLoggingArgs {
> enum Level {
> Info = 0,
> Warning = 1,
> Error = 2,
> Fatal = 3,
>
> }
> optional Level minloglevel = 1;
> optional string log_dir = 2;
> }
>
> —
> Reply to this email directly or view it on GitHub.